yb-voyager/src/srcdb/data/gather-assessment-metadata/postgresql/table-row-counts.psql

```diff
@@ -13,24 +13,24 @@ DECLARE
     schema_list TEXT[] := string_to_array(current_setting('vars.schema_list'), '|');
     schema_name TEXT;
     table_record RECORD;
-    sql_statement TEXT;
 BEGIN
     -- Iterate over each schema in the schema list
     FOREACH schema_name IN ARRAY schema_list LOOP
         -- Iterate over each table in the current schema
         FOR table_record IN 
-            SELECT table_schema, table_name
-            FROM information_schema.tables 
-            WHERE table_schema = schema_name AND table_type = 'BASE TABLE' 
+            SELECT s.relname AS table_name,
+                   n.nspname AS table_schema,
+                   COALESCE(NULLIF(s.n_live_tup, -1), 0) AS estimated_row_count
+            FROM pg_stat_user_tables s
```

Comment:
@sanyamsinghal pls verify that this is also supported by older PG versions (like 9)

Reply:
yeah `pg_stat_user_tables` is available even till PG7
https://www.postgresql.org/docs/9.3/monitoring-stats.html#:~:text=Table%2027%2D5.-,pg_stat_all_tables,-View